mac/ZDEFINE1.CPY

```diff
@@ -1,10 +1,22 @@
 .*********************************************************************
-.* Copyright 2007 Automated Software Tools Corporation               *
-.* This source code is part of z390 assembler/emulator package       *
-.* The z390 package is distributed under GNU general public license  *
-.* Author - Don Higgins                                              *
-.* Date   - 04/19/07                                                 *
+.* z390 - Mainframe assembler emulator and run-time engine
+.* Copyright (C) 2021 z390 Assembler LLC
+.*
+.* This file is part of z390.
+.*
+.* z390 is free software; you can redistribute it and/or modify
+.* it under the terms of the GNU General Public License as published by
+.* the Free Software Foundation; either version 2 of the License, or
+.* (at your option) any later version.
+.* z390 is distributed in the hope that it will be useful,
+.* but WITHOUT ANY WARRANTY; without even the implied warranty of
+.* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.* GNU General Public License for more details.
+.*
+.* You should have received a copy of the GNU General Public License 
+.* along with this program; if not, see https://www.gnu.org/licenses.
 .*********************************************************************
+.* 04/19/07 INITIAL CODING BY DON HIGGINS
 .* 09/02/07 RPI 681 DEFINE GLOBALS FOR DEFINE AND ZDEFINE MACROS     
 .* 09/23/07 RPI 704 ADD CONTROL INTERVAL SIZE FOR LINEAR SUPPORT     
```

Comment:
Among the lines above, there are many words shown in red. Not happening elsewhere. Not a problem (to me). Just pointing it out.

Reply:
So I think this is just the code highlighting which is assuming CPY files are for COBOL and trying to interpret it.
I would ignore the colour for these files.